cellprofiler/modules/unmixcolors.py

```diff
@@ -183,11 +183,13 @@ def create_settings(self):
         self.add_image(False)
```

Comment:
Just above this (at "Select the input color image", put a `\` after the `doc="""` and left-justify all text.

Reply:
how could I miss this =,=